snapcraft/parts/setup_assets.py

```diff
@@ -49,13 +49,16 @@ def setup_assets(
 
     _write_snap_directory(assets_dir=assets_dir, prime_dir=prime_dir, meta_dir=meta_dir)
 
+    # create wrappers for hooks in the snap/hooks directory
+    _create_hook_wrappers(prime_dir)
+
     if project.hooks:
```

Comment:
Add a code comment here stating that these will replace anything added by _create_hook_wrappers